docs/guide.md

```diff
@@ -0,0 +1,210 @@
+# Getting Started with PatternPack
+
+## Creating a New PatternPack
+The idea behind PatternPack is to extract the styles from your application and move them into a separate repository where they can be shared between one or multiple projects.
+
+You have multiple methods to generate a Pattern Library:
+
+- Build your own configuration (covered in this section)
+- Start with the [example library](https://github.com/patternpack/patternpack-example-library) ([documentation](https://github.com/patternpack/patternpack/blob/master/docs/getting-started.md#start-with-the-example-pattern-library))
```

Comment:
- Use the [example library](https://github.com/patternpack/patternpack-example-library) as a starting point for a new pattern library([documentation](https://github.com/patternpack/patternpack/blob/master/docs/getting-started.md#start-with-the-example-pattern-library))
- Use Yeoman Generator to create a new pattern library ([documentation](https://github.com/patternpack/patternpack/blob/master/docs/getting-started.md#use-the-yeoman-generators))
- Create a new pattern library manually (covered in this section)